PsychSourceGL/Source/Common/Base/PythonGlue/PsychScriptingGluePython.c

```diff
@@ -54,14 +54,25 @@
 // in February 2013:
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
```

Comment:
Maybe also increase from NumPy 1.7 api to 1.16, if we require that anyway?

Reply:
Sounds good, but I'll target 1.13 b/c that's the oldest supported numpy on Python 3.6 (https://github.com/scipy/oldest-supported-numpy/blob/main/setup.cfg#L53). Though it looks like targeting 1.13 & 1.16 should be functionally identical https://github.com/numpy/numpy/blob/50a74fb65fc752e77a2f9e9e2b7227629c2ba953/numpy/core/include/numpy/numpyconfig.h#L64